### PR TITLE
[Monit] Change the monitoring period from 120 seconds to 60 seconds.

### PR DESCRIPTION
**Pull request by yozhao101**
Signed-off-by: Yong Zhao <yozhao@microsoft.com>

**- What I did**

According to our offline discussion, we have decided to change the monitoring period
of Monit from 120 seconds to 60 seconds.

### CODE REVIEW

File: files/image_config/monit/monitrc [hunk 1]
```diff
@@ -16,7 +16,7 @@
 ##
 ## Start Monit in the background (run as a daemon):
 #
-  set daemon 120            # check services at 2-minute intervals
+  set daemon 60            # check services at 1-minute intervals
```
Comment (jleveque) [inline files/image_config/monit/monitrc]: Add extra space before comment to fix alignment

Comment (yozhao101) [reply files/image_config/monit/monitrc]: Ok, I will change all the existing SONiC monit configurations and add an extra space to fix alignment.

File: files/image_config/monit/conf.d/sonic-host [hunk 2]
```diff
@@ -6,17 +6,17 @@
 ###############################################################################
 
 check filesystem root-overlay with path /
-    if space usage > 90% for 5 times within 10 cycles then alert
+    if space usage > 90% for 5 times within 20 cycles then alert
```
Comment (jleveque) [inline files/image_config/monit/conf.d/sonic-host]: You doubled the number of cycles which makes the interval consistent with the previous interval, but you didn't doble the number of "hits" required to alert. You also need to double those to maintain similar behavior (e.g., `if space usage > 90% for 5 times within 10 cycles then alert` with 120 second cycles ~= `if space usage > 90% for 10 times within 20 cycles then alert` with 60 second cycles).

**Comment by yxieca** [comment]
retest vsimage please

**Pull request merged**